.expeditor/verify.pipeline.yml

```diff
@@ -199,23 +199,27 @@ steps:
     executor:
       docker:
         host_os: windows
-        os_version: 2016
         environment:
           - FORCE_FFI_YAJL=ext
           - CHEF_LICENSE=accept-no-persist
         shell: ["powershell", "-Command"]
 
 - label: "Functional Specs Windows :ruby: 2.6"
   commands:
-    - /workdir/scripts/bk_tests/bk_win_functional.ps1
+    - scripts/bk_tests/bk_win_functional.ps1
+  expeditor:
+    executor:
+      windows:
+        privileged: true
+        shell: ["powershell", "-Command"]
```

Comment:
i think you can get rid of `shell: ["powershell", "-Command"]`. that was only relevant when this was run in docker right? not relevant when running the job directly on the host.